drivers/sensor/sgp40/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_library_sources_ifdef(CONFIG_SGP40 sgp40.c)
```

Comment:
I'm guessing you're just copy/pasting the pattern from other sensor drivers, but this is redundant. Can you please just use:

```
zephyr_library_sources(sgp40.c)
```

I've sent a cleanup patch for the rest of the tree here, just so you know I'm not making a special case out of this driver: https://github.com/zephyrproject-rtos/zephyr/pull/36632

Reply:
You guessed right. I've updated this across all my pull requests :+1: